tests/sentry/workflow_engine/migration_helpers/test_migrate_rule_action.py

```diff
@@ -733,3 +733,160 @@ def test_github_action_migration_malformed(self):
 
         actions = build_notification_actions_from_rule_data_actions(action_data)
         assert len(actions) == 0
+
+    def test_azure_devops_migration(self):
+        action_data = [
```

Comment:
#Todo for a future date -- it'd be a nice to have to have a `self.create_notification_action` or something along those lines to help create these